astropy/coordinates/angles.py

```diff
@@ -405,12 +429,10 @@ def wrap_at(self, wrap_angle, inplace=False):
             return `None`.
```

Comment:
Return None or self?

Reply:
Not completely sure what you mean...  And I wondered why the inplace option wouldn't return `self`, but decided that I should try not to change the API in this PR (I already did more than perhaps I should have...)

Reply:
I just meant the documentation isn't complete about the return types.

Reply:
Still confused:
```
        out : Angle or `None`
```
This seems OK, no?

Reply:
My bad! When I clicked on the expand code, it was broken into two expansions, one of which I didn't see and which held the entire "Returns" sections.